cranelift-codegen/src/isa/x86/abi.rs

```diff
@@ -547,25 +635,28 @@ fn insert_common_epilogues(
     stack_size: i64,
     reg_type: ir::types::Type,
     csrs: &RegisterSet,
+    cfa_state: &CFAState,
 ) {
     while let Some(ebb) = pos.next_ebb() {
         pos.goto_last_inst(ebb);
         if let Some(inst) = pos.current_inst() {
             if pos.func.dfg[inst].opcode().is_return() {
-                insert_common_epilogue(inst, stack_size, pos, reg_type, csrs);
+                insert_common_epilogue(inst, stack_size, pos, reg_type, csrs, &cfa_state);
             }
         }
     }
 }
 
 /// Insert an epilogue given a specific `return` instruction.
 /// This is used by common calling conventions such as System V.
+/// TODO implement and handle _cfa_state more than one epilogue.
 fn insert_common_epilogue(
```

Comment:
Unlikely but important edge case, there is a `FrameLayoutChange` missing for the `pop rbp` added here. If `FrameLayoutChange` are used for unwinding, a signal delivered _at_ the following `ret`, it would unwind with the wrong frame pointer and probably cause the unwinder to abort or segfault. I think a `FrameLayoutChange::CallFrameAddressAt { reg: RU::rsp, as RegUnit, offset: 0 }` would be appropriate to refer to the current frame (eg the span starting at the return address, `[rsp]`), like the one for the function's initial layout